App exited unexpectedly

Notes
======
App exited unexpectedly - Travis build 1440

Environment
======
Environment Key | Value
:----: | :----:
version | 11.0
build | 175
UDID | 3c7d0770c6265875
Android Release | 5.1.1
Android SDK | 22
Brand | google
Device | flo
Log history
======
```java
4/25/17 7:56 AM  I/ViewModeFragment: doScrollToPosition: position=3, offset=0
4/25/17 7:56 AM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/25/17 7:56 AM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/25/17 7:55 AM  I/ViewModeFragment: doScrollToPosition: position=3, offset=0
4/25/17 7:55 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:55 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en_tit_text_reg
4/25/17 7:55 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo1_psa_text_reg
4/25/17 7:55 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo2_php_text_reg
4/25/17 7:55 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en_obs_text_obs
4/25/17 7:53 AM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/25/17 7:53 AM  I/com.door43.translationstudio.services.BackupService: en_obs_text_obs backed up
4/25/17 7:53 AM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/25/17 7:52 AM  I/ViewModeFragment: doScrollToPosition: position=3, offset=0
4/25/17 7:52 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:52 AM  I/com.door43.translationstudio.ui.dialogs.BackupDialog: The server contains conflicting changes for en_obs_text_obs
4/25/17 7:52 AM  I/PullTargetTranslationTask: Reverting to local license
4/25/17 7:52 AM  I/PullTargetTranslationTask: Reverting to server manifest
4/25/17 7:52 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:52 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:51 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:51 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:50 AM  I/ViewModeFragment: doScrollToPosition: position=3, offset=0
4/25/17 7:50 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:50 AM  I/com.door43.translationstudio.services.BackupService: Backups running every 5 minute/s
4/25/17 7:50 AM  I/com.door43.translationstudio.services.BackupService: starting backup service
4/25/17 7:50 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en_tit_text_reg
4/25/17 7:50 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo1_psa_text_reg
4/25/17 7:50 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo2_php_text_reg
4/25/17 7:50 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en_obs_text_obs
4/25/17 7:49 AM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/25/17 7:49 AM  I/com.door43.translationstudio.services.BackupService: en_obs_text_obs backed up
4/25/17 7:49 AM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/25/17 7:48 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:48 AM  I/com.door43.translationstudio.ui.dialogs.BackupDialog: The server contains conflicting changes for en_obs_text_obs
4/25/17 7:48 AM  I/PullTargetTranslationTask: Reverting to local license
4/25/17 7:48 AM  I/PullTargetTranslationTask: Reverting to server manifest
4/25/17 7:48 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:48 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:48 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:48 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:47 AM  I/com.door43.translationstudio.services.BackupService: Backups running every 5 minute/s
4/25/17 7:47 AM  I/com.door43.translationstudio.services.BackupService: starting backup service
4/25/17 7:47 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en_tit_text_reg
4/25/17 7:47 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo1_psa_text_reg
4/25/17 7:47 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo2_php_text_reg
4/25/17 7:46 AM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/25/17 7:46 AM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/25/17 7:46 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:46 AM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/25/17 7:46 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:46 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en_tit_text_reg
4/25/17 7:46 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo1_psa_text_reg
4/25/17 7:46 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo2_php_text_reg
4/25/17 7:46 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en_obs_text_obs
4/25/17 7:46 AM  I/com.door43.translationstudio.services.BackupService: en_obs_text_obs backed up
4/25/17 7:46 AM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/25/17 7:45 AM  I/ViewModeFragment: doScrollToPosition: position=3, offset=0
4/25/17 7:45 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:45 AM  I/com.door43.translationstudio.ui.dialogs.BackupDialog: The server contains conflicting changes for en_obs_text_obs
4/25/17 7:45 AM  I/PullTargetTranslationTask: Reverting to local license
4/25/17 7:45 AM  I/PullTargetTranslationTask: Reverting to server manifest
4/25/17 7:45 AM  I/com.door43.translationstudio.ui.dialogs.BackupDialog: SSH keys were registered with the server
4/25/17 7:45 AM  I/com.door43.translationstudio.ui.dialogs.BackupDialog: Authentication failed
4/25/17 7:45 AM  E/com.door43.translationstudio.tasks.PullTargetTranslationTask: git@git.door43.org:qa99/en_obs_text_obs.git: Auth fail
org.eclipse.jgit.api.errors.TransportException: git@git.door43.org:qa99/en_obs_text_obs.git: Auth fail
	at org.eclipse.jgit.api.FetchCommand.call(FetchCommand.java:139)
	at org.eclipse.jgit.api.PullCommand.call(PullCommand.java:261)
	at com.door43.translationstudio.tasks.PullTargetTranslationTask.pull(PullTargetTranslationTask.java:148)
	at com.door43.translationstudio.tasks.PullTargetTranslationTask.start(PullTargetTranslationTask.java:82)
	at org.unfoldingword.tools.taskmanager.ManagedTask.run(ManagedTask.java:78)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)
Caused by: org.eclipse.jgit.errors.TransportException: git@git.door43.org:qa99/en_obs_text_obs.git: Auth fail
	at org.eclipse.jgit.transport.JschConfigSessionFactory.getSession(JschConfigSessionFactory.java:159)
	at org.eclipse.jgit.transport.SshTransport.getSession(SshTransport.java:136)
	at org.eclipse.jgit.transport.TransportGitSsh$SshFetchConnection.<init>(TransportGitSsh.java:262)
	at org.eclipse.jgit.transport.TransportGitSsh.openFetch(TransportGitSsh.java:161)
	at org.eclipse.jgit.transport.FetchProcess.executeImp(FetchProcess.java:136)
	at org.eclipse.jgit.transport.FetchProcess.execute(FetchProcess.java:122)
	at org.eclipse.jgit.transport.Transport.fetch(Transport.java:1138)
	at org.eclipse.jgit.api.FetchCommand.call(FetchCommand.java:130)
	... 7 more
Caused by: com.jcraft.jsch.JSchException: Auth fail
	at com.jcraft.jsch.Session.connect(Session.java:512)
	at org.eclipse.jgit.transport.JschConfigSessionFactory.getSession(JschConfigSessionFactory.java:116)
	... 14 more

4/25/17 7:44 AM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/25/17 7:44 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:44 AM  I/com.door43.translationstudio.services.BackupService: en-x-demo2_php_text_reg backed up
4/25/17 7:44 AM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/25/17 7:44 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:44 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:44 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:44 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:44 AM  I/ViewModeFragment: doScrollToPosition: position=13, offset=0
4/25/17 7:43 AM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/25/17 7:43 AM  I/com.door43.translationstudio.services.BackupService: Backups running every 5 minute/s
4/25/17 7:43 AM  I/com.door43.translationstudio.services.BackupService: starting backup service
4/25/17 7:43 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en_obs_text_obs
4/25/17 7:43 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en_tit_text_reg
4/25/17 7:43 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo1_psa_text_reg
4/25/17 7:43 AM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo2_php_text_reg
4/24/17 6:08 PM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/24/17 6:07 PM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/24/17 6:07 PM  I/ViewModeFragment: doScrollToPosition: position=13, offset=0
4/24/17 6:07 PM  I/ViewModeFragment: doScrollToPosition: position=13, offset=0
4/24/17 5:15 PM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/24/17 5:15 PM  I/com.door43.translationstudio.services.BackupService: en-x-demo2_php_text_reg backed up
4/24/17 5:15 PM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/24/17 5:15 PM  I/ViewModeFragment: doScrollToPosition: position=13, offset=0
4/24/17 4:32 PM  E/com.door43.translationstudio.core.TargetTranslation: Failed to commit changes for en-x-demo2_php_text_reg
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at org.eclipse.jgit.api.CommitCommand.call(CommitCommand.java:203)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1069)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1034)
	at com.door43.translationstudio.core.TargetTranslation$1.run(TargetTranslation.java:1108)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_php_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 4 more

4/24/17 4:32 PM  E/TargetTranslation: Failed to stage changes for en-x-demo2_php_text_reg
org.eclipse.jgit.api.errors.JGitInternalException: Exception caught during execution of add command
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:212)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1051)
	at com.door43.translationstudio.core.TargetTranslation.commitSync(TargetTranslation.java:1034)
	at com.door43.translationstudio.core.TargetTranslation$1.run(TargetTranslation.java:1108)
Caused by: org.eclipse.jgit.errors.LockFailedException: Cannot lock /storage/emulated/0/translationStudio/translations/en-x-demo2_php_text_reg/.git/index
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:224)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:301)
	at org.eclipse.jgit.dircache.DirCache.lock(DirCache.java:267)
	at org.eclipse.jgit.lib.Repository.lockDirCache(Repository.java:1052)
	at org.eclipse.jgit.api.AddCommand.call(AddCommand.java:142)
	... 3 more

4/24/17 1:21 PM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/24/17 1:18 PM  I/com.door43.translationstudio.services.BackupService: en-x-demo2_php_text_reg backed up
4/24/17 1:16 PM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/24/17 1:15 PM  I/ViewModeFragment: doScrollToPosition: position=13, offset=0
4/24/17 1:13 PM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/24/17 1:10 PM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/24/17 1:09 PM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/24/17 1:09 PM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/24/17 1:09 PM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/24/17 1:09 PM  I/com.door43.translationstudio.services.BackupService: en-x-demo2_php_text_reg backed up
4/24/17 1:09 PM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/24/17 1:08 PM  I/ViewModeFragment: doScrollToPosition: position=13, offset=0
4/24/17 1:08 PM  I/ViewModeFragment: doScrollToPosition: position=13, offset=0
     

```

--- a/app/src/main/java/com/door43/translationstudio/ui/dialogs/BackupDialog.java
+++ b/app/src/main/java/com/door43/translationstudio/ui/dialogs/BackupDialog.java
@@ -873,7 +873,7 @@ public class BackupDialog extends DialogFragment implements SimpleTaskWatcher.On
             args.putInt(App.EXTRA_VIEW_MODE, TranslationViewMode.REVIEW.ordinal());
             intent.putExtras(args);
             startActivity(intent);
-            getActivity().finish();
+            BackupDialog.this.dismiss();
         }
     }
 

